src/Engine/Objects/NPC.h

```diff
@@ -28,8 +28,11 @@ bool CheckHiredNPCSpeciality(NpcProfession prof);
 int UseNPCSkill(NpcProfession profession, int id);
 
 const std::string &GetProfessionActionText(NpcProfession prof);
-NPCData *GetNPCData(signed int npcid);
-NPCData *GetNewNPCData(signed int npcid, int *npc_indx);
+
+/**
+ * @offset 0x445B2C
+ */
+NPCData *getNPCData(signed int npcid, int *npc_indx = nullptr);
```

Comment:
Let's add docs here while we're touching this code? I honestly have no idea what's the semantic behind `npcid` and `npc_indx`.

Reply:
After looking a little more I think that `npc_indx` can be removed. It's needed only once and can be replaced with more straightforward code.